### ISSUE TITLE
Is it correct to take only last day rows for test evaluations? 

### ISSUE THREAD

**Issue by Zillibub** [geekidharsh/predicting-harddrive-failures-using-ml#1]
`test = hdd_last_day.query('index in @test_ids')`
If you are creating test data set only from last rows, can it be data leak? 
In production you don't know which day will be last for current disk.

**Comment by geekidharsh** [comment]
Yes, that could be a data leak, available dataset I used is incomplete. Since this is mainly a test exercise I didn't worry too much about possible data leak in this situation.
